### PR TITLE
[Doc EN]: `guide/async-data.md`

### PR DESCRIPTION
**Pull request by Haeresis**
In our French translation/review we have changed some consistent things, so we suggest you do the same in the original documentation.

More details in comments.

Do not hesitate to set back the original behavior for some changes if you think they do not make sense.

### CODE REVIEW

File: en/guide/async-data.md [hunk 1]
```diff
@@ -8,10 +8,7 @@ Nuxt.js adds an `asyncData` method to let you handle async operations before set
 
 ## The asyncData Method
 
-Sometimes you just want to fetch data and pre-render it on the server-side without using a store.
-`asyncData` is called every time before loading the component (**only for pages components**).
-It can be called server-side or before navigating to the corresponding route.
-This method receives [the context](/api/context) as the first argument, you can use it to fetch some data and nuxt.js will merge it with the component data.
+Sometimes you just want to fetch data and pre-render it on the server-side without using a store. `asyncData` is called every time before loading the component (**only for pages components**). It can be called server-side or before navigating to the corresponding route. This method receives [the context](/api/context) as the first argument, you can use it to fetch some data and Nuxt.js will merge it with the component data.
```
Comment (Haeresis) [inline en/guide/async-data.md]: Line return not useful because the render will be on one line.

File: en/guide/async-data.md [hunk 2]
```diff
@@ -22,6 +19,7 @@ Nuxt.js offers you different ways to use `asyncData`. Choose the one you're the
 3. Define a callback as second argument. It has to be called like this: `callback(err, data)`
 
 ### Returning a Promise
+
```
Comment (Haeresis) [inline en/guide/async-data.md]: Markdown one line empty convention

File: en/guide/async-data.md [hunk 3]
```diff
@@ -34,6 +32,7 @@ export default {
 ```
 
 ### Using async/await
+
```
Comment (Haeresis) [inline en/guide/async-data.md]: Markdown one line empty convention

File: en/guide/async-data.md [hunk 4]
```diff
@@ -44,6 +43,7 @@ export default {
 ```
 
 ### Using a callback
+
```
Comment (Haeresis) [inline en/guide/async-data.md]: Markdown one line empty convention

File: en/guide/async-data.md [hunk 5]
```diff
@@ -68,9 +68,9 @@ You can display the data inside your template like you're used to doing:
 
 ## The Context
 
-To see the list of available keys in `context`, take a look at the [API Pages data](/api).
+To see the list of available keys in `context`, take a look at the [API Essential `context`](/api/context).
```
Comment (Haeresis) [inline en/guide/async-data.md]: The correct section about keys in `context` is this one.

File: en/guide/async-data.md [hunk 6]
```diff
@@ -79,6 +79,7 @@ You can use the context object injected into the `asyncData` property to access
 Nuxt.js adds the `error(params)` method in the `context`, you can call it to display the error page. `params.statusCode` will be also used to render the proper status code form the server-side.
 
 Example with a `Promise`:
+
```
Comment (Haeresis) [inline en/guide/async-data.md]: Markdown one line empty convention

File: en/guide/async-data.md [hunk 7]
```diff
@@ -93,7 +94,8 @@ export default {
 }
 ```
 
-If you're using the `callback` argument, you can call it directly with the error and nuxt.js will call the `error` method for you:
+If you're using the `callback` argument, you can call it directly with the error and Nuxt.js will call the `error` method for you:
+
```
Comment (Haeresis) [inline en/guide/async-data.md]: Markdown one line empty convention

File: en/guide/async-data.md [hunk 8]
```diff
@@ -108,4 +110,4 @@ export default {
 }
 ```
 
-To customize the error page, take a look at the [VIEWS layouts section](/guide/views#layouts).
+To customize the error page, take a look at the [Guide Views `layouts`](/guide/views#layouts).
```
Comment (Haeresis) [inline en/guide/async-data.md]: To reason for all capitalise Word. To keep consistency with others link. I propose to present link in the format « Section Page SectionPage » like others link (e.g. API Pages `asyncData`) or (API Essential `context`)

File: en/guide/async-data.md [hunk 9]
```diff
@@ -108,4 +110,4 @@ export default {
 }
 ```
 
-To customize the error page, take a look at the [VIEWS layouts section](/guide/views#layouts).
+To customize the error page, take a look at the [Guide Views layouts](/guide/views#layouts).
```
Comment (Haeresis) [inline en/guide/async-data.md]: To reason for all capitalize Word. To keep consistency with others link. I propose to present link in the format « Section Page SectionPage » like others link (e.g. API Pages `asyncData`) or (API Essential `context`)

File: en/guide/async-data.md (hunk 8 above)
Comment (Haeresis) [inline en/guide/async-data.md]: To reason for all capitalize Word. To keep consistency with others link. I propose to present link in the format « Section Page SectionPage » like others link (e.g. API Pages `asyncData`) or (API Essential `context`)

**Comment by Haeresis** [comment]
@Atinux, I suggest you this fourth PR on five.

**Comment by Atinux** [comment]
:raised_hands:

**Pull request merged**